### ISSUE TITLE
Some element are not showing and overlaps with each other

### ISSUE THREAD

**Issue by billyromanooo** [nimsandu/spicetify-bloom#306]
### Did you read the README?

- [X] I have read the README, the theme is installed/configured correctly and the "Troubleshooting" section contains no information related to my problem

### Do you have the latest versions of Spicetify, Spotify and Bloom?

- [X] I'm using the latest versions of Spicetify, Spotify and Bloom

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Is your problem definitely caused by the theme?

- [X] I have checked the app behavior without the theme and it is different

### Environment & Computer Info

```Markdown
- Spotify version: Spotify for Windows (64 bit) 1.2.25.1011.g0348b2ea
- Installed from: Spotify site
- Spicetify version: Spicetify v2.28.0
- Color scheme: Bloom / darkmono
- Extensions: webnowplaying.js, adblock.js, oneko.js, volumePercentage.js, playlist-icons.js, romaji-convert.js, queuePanel.js, imaged-folders.js, playNextWrapper.js, theme.js
- Custom apps: marketplace, betterLibrary, eternal-jukebox, better-local-files
- Bloom installation method: Spicetify Marketplace
```

### Description

1. Show the lyrics button missing
![1](https://github.com/nimsandu/spicetify-bloom/assets/32069026/17498377-589f-4de0-a1fd-2e19cb924081)
2. The profile picture overlaps with the minimize button 
![2](https://github.com/nimsandu/spicetify-bloom/assets/32069026/a6bbc74e-d7b9-4e62-9756-5750d4285003)
3. Pause button overlaps with now playing GIF 
![3](https://github.com/nimsandu/spicetify-bloom/assets/32069026/9e46a2e2-3b4d-48d5-b353-7813a8106928)
4. Incorrect library and local files icon
![4](https://github.com/nimsandu/spicetify-bloom/assets/32069026/74dadb9a-5893-423b-9134-920a98006921)
5. The previous button is little bit higher then it supposed to
![5](https://github.com/nimsandu/spicetify-bloom/assets/32069026/591b0f6f-0c76-4355-97df-6eaf49ce0bce)

**Comment by francescoaracu** [comment]
I add another one: when I hover on a searched song, heart and play icons are overlapped. This happens only if the snippet "Left aligned heart icons" is activated.

![image](https://github.com/nimsandu/spicetify-bloom/assets/48188510/9d58df7a-5238-43f0-9356-1f2b0b8786d3)

**Comment by TheChilledBuffalo** [comment]
Bugs 3 and 4 were fixed in [#308](https://github.com/nimsandu/spicetify-bloom/pull/308)

**Comment by TheChilledBuffalo** [comment]
After recent updates, "Your Library" is not in the same menu as "Home" and "Search". 

@billyromanooo May I know, which extension you used to get "Your Library" and "Local Files" in that menu?

**Comment by SunsetTechuila** [comment]
@TheChilledBuffalo https://github.com/Sowgro/betterLibrary and https://github.com/Pithaya/spicetify-apps-dist/tree/dist/better-local-files

**Comment by TheChilledBuffalo** [comment]
Thanks! I'll try and fix it asap.

**Comment by SunsetTechuila** [comment]
- Pause button overlaps with now playing GIF - fixed
 - Incorrect library and local files icon - fixed
 - The previous button is little bit higher then it supposed to - fixed with e84cb64
 - Show the lyrics button missing - not a theme bug, spotify moved lyrics to the right sidebar
 - heart and play icons are overlapped with the snippet "Left aligned heart icons" - not going to fix
 - The profile picture overlaps with the minimize button - difficult to fix, will not be fixed in the near future
